src/maps/india/rivers.tsx

```diff
@@ -0,0 +1,64 @@
+import {ReactNode} from "react";
+
+export const RIVERS: ReactNode = <>
```

Comment:
Same comment as before, should we put this in a function?

Reply:
Yup, I'll make that update here.